integrations/kube-agent-updater/pkg/podutils/filter.go

```diff
@@ -63,7 +63,7 @@ func Not(filterFunc FilterFunc) FilterFunc {
 	}
 }
 
-const podReadinessGracePeriod = 10 * time.Minute
+const podReadinessGracePeriod = 5 * time.Minute
```

Comment:
We should probably leave a comment if we end up changing this value in the future